diff --git a/dist/node/chunks/dep-CzJTQ5q7.js b/dist/node/chunks/dep-CzJTQ5q7.js
index ae55745c78a34eaf9bca42a9c1b82cca0d04086c..1bfb706d480a6cd7241db299782f48a4f33864b9 100644
--- a/dist/node/chunks/dep-CzJTQ5q7.js
+++ b/dist/node/chunks/dep-CzJTQ5q7.js
@@ -66746,16 +66746,6 @@ assetFileNames isn't equal for every build.rollupOptions.output. A single patter
       colors$1.yellow(`
 (!) Experimental legacy.buildSsrCjsExternalHeuristics and ssr.format were be removed in Vite 5.
     The only SSR Output format is ESM. Find more information at https://github.com/vitejs/vite/discussions/13816.
-`)
-    );
-  }
-  const resolvedBuildOutDir = normalizePath$3(
-    path$m.resolve(resolved.root, resolved.build.outDir)
-  );
-  if (isParentDirectory(resolvedBuildOutDir, resolved.root) || resolvedBuildOutDir === resolved.root) {
-    resolved.logger.warn(
-      colors$1.yellow(`
-(!) build.outDir must not be the same directory of root or a parent directory of root as this could cause Vite to overwriting source files with build outputs.
 `)
     );
   }
